plugin/Field/CustomLabel/CustomLabelCreate.php

```diff
@@ -77,18 +77,18 @@ public function createForField(CustomLabelModelField $pDataModel): int
 
 	/**
 	 *
-	 * @param CustomLabelModelBase $pDataModel
+	 * @param CustomLabelModelField $pDataModel
 	 * @return int
 	 *
 	 * @throws RecordManagerInsertException
 	 *
 	 */
 
-	private function createBase(CustomLabelModelBase $pDataModel): int
+	private function createBase(CustomLabelModelField $pDataModel): int
 	{
 		$field = $pDataModel->getField()->getName();
 		$customsLabelsId = $this->writeDatabaseGeneral($pDataModel->getFormId(), $field);
-		$pDataModel->setCustomsLabelsId($customsLabelsId);
+		$pDataModel->withCustomLabelsId($customsLabelsId);
 		return $customsLabelsId;
```

Comment:
`$pDataModel->withCustomLabelsId($customsLabelsId);` should be returned here, right?